mlflow/store/artifact/mlflow_artifacts_repo.py

```diff
@@ -41,8 +41,10 @@ def _validate_uri_scheme(scheme):
 class MlflowArtifactsRepository(HttpArtifactRepository):
     """Scheme wrapper around HttpArtifactRepository for mlflow-artifacts server functionality"""
 
-    def __init__(self, artifact_uri):
-        super().__init__(self.resolve_uri(artifact_uri, get_tracking_uri()))
```

Comment:
Here is the main issue. The `HttpArtifactRepository` repo is initialized with `get_tracking_uri()` and does not pass the tracking URI from the client.